packages/features/ee/dsync/components/GroupNameCell.tsx

```diff
@@ -0,0 +1,99 @@
+import { useState } from "react";
+
+import { classNames } from "@calcom/lib";
+import { trpc } from "@calcom/trpc/react";
+import { Badge, TextField, showToast } from "@calcom/ui";
+import { X, Plus } from "@calcom/ui/components/icon";
+
+const GroupNameCell = (props) => {
+  const [groupNames, setGroupNames] = useState(props.groupNames);
```

Comment:
Thoughts on using state here? As long as the mutation is successful let's update what's already rendered so we don't have to requery for group names and reduce the payload that's sent back.

Reply:
Props is inferred as any